src/quo/components/utilities/token/view.cljs

```diff
@@ -33,6 +35,19 @@
 
 (def ^:private b64-png-image-prefix "data:image/png;base64,")
 
+(defn temp-empty-symbol
+  [size]
+  [rn/view
+   {:style (merge (token-style {:justify-content :center
+                                :align-items     :center
+                                :border-radius   20
+                                :border-width    1
+                                :border-color    :grey}
+                               size))}
+   [quo/text
+    {:size  :paragraph-2
+     :style {:color :grey}} "TBA"]])
+
```

Comment:
Temporary label for tokens that we don't have their icon

Reply:
I believe the fallback is the tokens initials (confirm with design) can we handle that here?
Cc @ulisesmac

Reply:
Thanks! 🙏

Reply:
Yep, they said it's going to be a letter but there are no designs yet @J-Son89